src/System Application/App/Permission Sets/src/PermissionSetLookupList.Page.al

```diff
@@ -0,0 +1,87 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.Security.AccessControl;
+
+/// <summary>
+/// Look up page for selecting a permission set from Tenant Permission Set or Metadata Permission Set.
+/// </summary>
+page 9878 "Permission Set Lookup List"
+{
+    Caption = 'Permission Set Lookup';
+    Editable = false;
+    DeleteAllowed = false;
```

Comment:
No need to disable delete, insert and modify if the page isn't editable :)

Reply:
Removed